src/H5detect.c

```diff
@@ -205,12 +205,15 @@ precision(detected_t *d)
 }
 
 /*-------------------------------------------------------------------------
- * Function:    DETECT_I
+ * Function:    DETECT_I/DETECT_BYTE
  *
- * Purpose:     This macro takes a type like `int' and a base name like
+ * Purpose:     These macro takes a type like `int' and a base name like
```

Comment:
This instead of These

Reply:
Yes, I missed it when I reverted, then left it for an eventual fix that accomodates both big-endian and OpenBSD.